django_cas_ng/__init__.py

```diff
@@ -15,6 +15,7 @@
     'CAS_RENEW': False,
     'CAS_IGNORE_REFERER': False,
     'CAS_LOGOUT_COMPLETELY': True,
+    'CAS_FORCE_LOWERCASE': False,
```

Comment:
Can you add username as part of settings, to make it more clear. e.g. 
CAS_FORCE_CHANGE_USERNAME_CASE = 'lower' 
or
CAS_FORCE_CHANGE_USERNAME_CASE = 'upper'
then it can be config as lowercase or upper case, if want to most flexible, the config value can be a function to change username on the fly.